Gruntfile.js

```diff
@@ -613,7 +613,11 @@ module.exports = function(grunt) {
       }
     },
 
-    uglify: {
+    'build-js': {
+      command: 'npm run build-js'
```

Comment:
Why are we calling npm scripts from Grunt? Aren't we trying to move things out of Grunt and use npm directly?